snowpack/src/scan-imports.ts

```diff
@@ -235,32 +247,11 @@ export async function scanImports(cwd: string, config: SnowpackConfig): Promise<
   const loadedFiles: (SnowpackSourceFile | null)[] = await Promise.all(
     includeFiles.map(async (filePath) => {
       const {baseExt, expandedExt} = getExt(filePath);
-      // Always ignore dotfiles
-      if (filePath.startsWith('.')) {
-        return null;
-      }
-
-      // Probably a license, a README, etc
-      if (baseExt === '') {
-        return null;
-      }
-
-      // If we don't recognize the file type, it could be source. Warn just in case.
-      if (!Object.keys(srcFileExtensionMapping).includes(baseExt) && !mime.lookup(baseExt)) {
-        logger.warn(
-          colors.dim(
-            `ignoring unsupported file "${path
-              .relative(process.cwd(), filePath)
-              .replace(/\\/g, '/')}"`,
-          ),
-        );
-      }
-
       return {
         baseExt,
         expandedExt,
         locOnDisk: filePath,
-        contents: await fs.promises.readFile(filePath, 'utf-8'),
```

Comment:
The underlying issue: we were trying to enforce binary files be code, which led to breakages when we tried to parse that nonsense code